lib/core/controlplane_api_direct.rb

```diff
@@ -98,7 +98,7 @@ def should_refresh_api_token?
   end
 
   def refresh_api_token
-    @@api_token[:token] = `cpln profile token`.chomp
+    @@api_token[:token] = Shell.cmd("cpln", "profile", "token")[:output].chomp
```

Comment:
The backtick in `run.rb` should not be replaced because it has to run in a terminal environment.

There's a note there explaining it:

```
# NOTE: cannot use `Shell.cmd` here, as `stty size` has to run in a terminal environment
```